neura/chain.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "neura",
+  "chain_id": "neura_266-1",
+  "bech32_prefix": "neura",
```

Comment:
Please register this prefix here:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
https://github.com/satoshilabs/slips/pull/1733